### DISCUSSION TITLE
Conversion to EU spec and maps etc

### DISCUSSION THREAD

**Discussion by NMarq** [Mr-MIBonk/M.I.B._More-Incredible-Bash#279] [Q&A]
Hi,

We a Porsche with firmware MHI2_JP_POG11_P0816 0816 is this patchable/convertable to EU spec and maps?

Thanks
Nick

**Comment by harman-f** [comment]
Yes, it is. 
read up on mibwiki.one 
There is a function integrated into M.I.B
